src/stampit.js

```diff
@@ -6,80 +6,76 @@
  * Copyright (c) 2013 Eric Elliott
  * http://opensource.org/licenses/MIT
  **/
-'use strict';
-var forEach = require('lodash/collection/forEach');
-var map = require('lodash/collection/map');
-var forOwn = require('lodash/object/forOwn');
-var isFunction = require('lodash/lang/isFunction');
-var isArray = Array.isArray;
-var isObject = require('lodash/lang/isObject');
-var create = Object.create;
-var slice = require('lodash/array/slice');
-var mixer = require('supermixer');
+import forEach from 'lodash/collection/forEach';
+import forOwn from 'lodash/object/forOwn';
+import isFunction from 'lodash/lang/isFunction';
+import isObject from 'lodash/lang/isObject';
+import map from 'lodash/collection/map';
+import {
+  merge,
+  mergeChainNonFunctions,
+  mergeUnique,
+  mixin,
+  mixinChainFunctions,
+  mixinFunctions,
+} from 'supermixer';
+
+const create = Object.create;
 
 /* jshint -W024 */
 
 // Avoiding JSHist W003 violations.
-var stampit;
+let stampit;
```

Comment:
Consider `var` deprecated. I can't think of a good reason to use `var` instead of `const` or `let`.

Prefer `const` whenever possible.